main.go

```diff
@@ -189,7 +189,7 @@ func main() {
 
 	if cfg.EnableRepoClone {
 		handleStepError := func(stepID, tag string, err error, shortMsg string) {
-			LogError(stepID, tag, err, shortMsg)
+			LogError(stepID, tag, err, shortMsg) //nolint:govet
```

Comment:
What is the issue we are ignoring here?

Reply:
The signature of `LogError()` is `format string, v ...interface{}`, and the fmt placeholders in `format` can't be statically analyzed because it comes from a string variable (`shortMsg`) and not a static string.

Since the whole purpose of this function is to print dynamic step errors, I think it's okay and I couldn't find an alternative.